src/client/app/components/UIOptionsComponent.jsx

```diff
@@ -93,14 +132,50 @@ export default class UIOptionsComponent extends React.Component {
 						<div className="checkbox">
 							<label><input type="checkbox" onChange={this.handleChangeBarStacking} checked={this.props.barStacking} />Bar stacking</label>
 						</div>
-						<p style={labelStyle}>Bar chart interval (days):</p>
-						<Slider min={1} max={365} value={this.state.barDuration} onChange={this.handleBarDurationChange} onChangeComplete={this.handleBarDurationChangeComplete} />
+						<p style={labelStyle}>Bar chart interval:</p>
+						<ToggleButtonGroup
```

Comment:
Recommend refactoring this. I want this done soon, so it is not a good first issue.